lib/features/composer/presentation/composer_controller.dart

```diff
@@ -2235,6 +2235,18 @@ class ComposerController extends BaseController
     }
   }
 
+  bool _validateCloseComposerWithoutSave() {
+    if (composerArguments.value == null) return true;
+
+    if (PlatformInfo.isWeb &&
+        !_isEmailBodyLoaded &&
+        !screenDisplayMode.value.isNotContentVisible()) return true;
```

Comment:
with it, save will be called all the time. please make sure, user dont need to `save` if nothing change

Reply:
It's not a condition to save, just a condition to be able to close immediately or not. The condition you mentioned is below.

Reply:
https://github.com/user-attachments/assets/d1a5ee94-6919-4db8-809f-bea4b43f39d5

Reply:
Fixed


https://github.com/user-attachments/assets/57082302-cfb0-4f98-81ff-55bb0e7d4779